docs/apache-airflow/howto/define-extra-link.rst

```diff
@@ -31,7 +31,7 @@ The following code shows how to add extra links to an operator via Plugins:
 .. code-block:: python
 
     from airflow.models.baseoperator import BaseOperator
-    from airflow.models.baseoperatorlink import BaseOperatorLink
+    from airflow.sdk.definitions.baseoperatorlink import BaseOperatorLink
```

Comment:
We should probably add an alias and promote `from airflow.sdk import BaseOperatorLink` instead.

By the way, should be import to BaseOperator above be changed too?

Reply:
It should yes.

```suggestion
    from airflow.sdk import BaseOperator, BaseOperatorLink
```

Reply:
@amoghrajesh Important rule to follow: User facing docs should instruct people to only import from `airflow.sdk`, never directly from `.definitions` please.

Reply:
Oh thanks. Yeah, let me make that change.
Will keep it in mind too.

Reply:
> By the way, should be import to BaseOperator be changed too?

Yeah I guess that needs to be done too. The porting is however done